data/ray-tracing/ray-tracing_parameters.yml

```diff
@@ -0,0 +1,13 @@
+zenithAngle: 
+  len: 1
+  unit: !astropy.units.Unit {unit: deg}
+  default: 20
+  names: ['zenith', 'theta']
+offAxisAngle: 
+  len: null
```

Comment:
What is the meaning of the 'len' parameter? Why is it null for the offAxisAngle, but '1' for the other parameters?

Reply:
The explanations can be found as comments in test_parameters.yml. I assume you found them.